pyproject.toml

```diff
@@ -1,49 +1,49 @@
-[tool.poetry]
+[project]
 name = "iceaxe"
-version = "0.1.0"
+version = "0.5.4"
```

Comment:
This should stay on 0.1.0: we bump automatically in CI

Reply:
![image](https://github.com/user-attachments/assets/4c1d7553-0246-497d-823e-3bc999566ba6)

As far as I can tell, that was the case before the switch to `uv` (see image), but `uv` doesn't have an equivalent to `poetry version` as far as I can tell. We can use `sed` or just add a check like https://github.com/pydantic/pydantic-ai/blob/fd90fc6af5b129cd576568df34d6927cecb5fded/.github/workflows/ci.yml#L319

Reply:
I have a script that I use for Mountaineer that works pretty well: https://github.com/piercefreeman/mountaineer/blob/main/.github/scripts/update_version.py. For now how about we clone that into the CI environment and run it against iceaxe? Eventually I need to get around to refactoring that bump logic into its own action.

Reply:
@piercefreeman The script has bee added.